base_source.py

```diff
@@ -0,0 +1,137 @@
+# import json
+import os
+import shutil
+import tempfile
+
+import requests
+
+from data_retrieval.files import copy_file_to_s3
+
+OPEN_HUMANS_TOKEN_REFRESH_URL = os.getenv(
+    'OPEN_HUMANS_TOKEN_URL',
+    'https://www.openhumans.org/api/processing/refresh-token/')
+
+PRE_SHARED_KEY = os.getenv('PRE_SHARED_KEY')
```

Comment:
Add a comment noting that the PRE_SHARED_KEY is shared between data processing and the main site, to authenticate them to each other?